Library/Homebrew/attestation.rb

```diff
@@ -63,7 +66,7 @@ def self.gh_executable
       #       to prevent a cycle during bootstrapping. This can eventually be resolved
       #       by vendoring a pure-Ruby Sigstore verifier client.
       @gh_executable ||= T.let(with_env("HOMEBREW_VERIFY_ATTESTATIONS" => nil) do
```

Comment:
While we're at it: let's flip this to:

```suggestion
      @gh_executable ||= T.let(with_env("HOMEBREW_NO_VERIFY_ATTESTATIONS" => "1") do
```